src/app/login/page.tsx

```diff
@@ -0,0 +1,145 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import arrow from '@/assets/images/arrow.svg';
+import Link from 'next/link';
+import LoginButton from '@/components/login/LoginBtn';
+import { useRouter } from 'next/navigation';
+import { loginUser } from '@/api/requests';
+import { useCookies } from 'react-cookie';
+import { userInfoState } from '@/atom/states';
+import { useRecoilState } from 'recoil';
+
+export default function page() {
+  const router = useRouter();
+  const [cookies, setCookie, removeCookie] = useCookies();
+  const [userInfo, setUserInfo] = useRecoilState(userInfoState);
+  const [id, setId] = useState('');
+  const [password, setPassword] = useState('');
+  const isAllFieldsFilled = !!(id.trim() && password);
+  const handleIdChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setId(e.target.value);
+  };
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
+  const loginHandler = async (e: { preventDefault: () => void }) => {
+    e.preventDefault();
+    if (!isAllFieldsFilled) {
+      alert('로그인 및 비밀번호를 입력해주세요');
+    } else {
+      const response = await loginUser(id, password);
+      if (response.success) {
+        // 로그인 성공
+        setUserInfo({
+          id: response.data.user.id,
+          part: response.data.user.part,
+          team: response.data.user.team,
+          userName: response.data.user.username,
+          userId: response.data.user.userid,
+        });
```

Comment:
이부분은 
```suggestion
        setUserInfo(response.data.user);
```
와 같이 하고 필요한 data만 꺼내쓰는식으로 해도 될것같다는 생각이 드네요~!